<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Issac" failures="1" tests="1" name="Requisition Reject" time="4.424" errors="0" timestamp="2025-04-30T14:12:07 IST">
  <testcase classname="com.source.classes.requisitions.reject.RejectTest" name="reject" time="1.249">
    <failure type="java.lang.AssertionError" message="Requisition Edit was not successful expected [0] but found [200]">
      <![CDATA[java.lang.AssertionError: Requisition Edit was not successful expected [0] but found [200]
at com.source.classes.requisitions.reject.RejectTest.reject(RejectTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- reject -->
</testsuite> <!-- Requisition Reject -->
